gateway/dapiimpl/proxy/proxy.go

```diff
@@ -203,6 +207,36 @@ func (p *DataApiProxy) proxyService(
 	newUrl, _ := url.Parse(svcEndpoint + relPath)
 	newUrl.RawQuery = r.URL.RawQuery
 
+	if p.disableAdmin {
+		switch serviceName {
+		case ServiceTypeMgmt:
+			if r.Method != http.MethodGet {
+				p.writeError(w, err, "admin endpoints are disabled")
+				return
+			}
+		case ServiceTypeQuery:
+			if !strings.HasPrefix(relPath, "/query") {
+				p.writeError(w, err, "admin endpoints are disabled")
+				return
+			}
+		case ServiceTypeSearch:
+			if !strings.HasPrefix(relPath, "/api/index") &&
```

Comment:
This would seem to allow defining indexes and deleting them per what I read in the docs.  Is that right?

Reply:
Correct.

Reply:
Do we need to prevent POST/PUT operations on those endpoints?